### PR TITLE
Add: Page for <noscript> for courses/:id

### PR DESCRIPTION
**Pull request by firefinchdev**

### CODE REVIEW

**Comment by firefinchdev** [comment]
Solves #3 
@abhishek97 Pls Review.

File: routes/courses.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+  let id = req.params.id
+
+  let dataFetch = {}
+  await getAPIdata(config.API.COURSE_URL + id)
+    .then((data) => {
+      dataFetch.course = deserializer(data)
+    }).catch((e) => {
+      console.log(e);
+      res.send('fuck')
```
Comment (abhishek97) [inline routes/courses.js]: remove this

File: routes/courses.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+var route = express.Router();
+
+
+route.get('/:id', async (req, res, next) => {
+
+  let id = req.params.id
+
+  let dataFetch = {}
+  await getAPIdata(config.API.COURSE_URL + id)
+    .then((data) => {
```
Comment (abhishek97) [inline routes/courses.js]: Inside an async function try to use await as much as possible without changing the meaning.
Here both the courses endpoint and recommended courses endpoint can be fetched parallelly, wrap the promises inside a Promise.all()

**Comment by firefinchdev** [comment]
@abhishek97 Changes commited.

**Comment by abhishek97** [comment]
Fix Nav menu, it should appear on top:
screen shot 2018-07-19 at 12 36 55 pm

Center Align these:

screen shot 2018-07-19 at 12 37 02 pm

**Comment by firefinchdev** [comment]
@abhishek97 
Done.

**Pull request merged**